Grbl_Esp32/src/GCode.cpp

```diff
@@ -865,14 +926,16 @@ uint8_t gc_execute_line(char* line, uint8_t client) {
                 FAIL(STATUS_GCODE_UNDEFINED_FEED_RATE);  // [Feed rate undefined]
             }
             switch (gc_block.modal.motion) {
-                case MOTION_MODE_LINEAR:
+                case Motion::None: break;  // Feed rate is unnecessary
+                case Motion::Seek: break;  // Feed rate is unnecessary
+                case Motion::Linear:
                     // [G1 Errors]: Feed rate undefined. Axis letter not configured or without real value.
                     // Axis words are optional. If missing, set axis command flag to ignore execution.
                     if (!axis_words)
```

Comment:
braces {.. }

Don't mind all the braces comments, I also use it to keep track where I am with this tough file... :)